packages/dx-react-grid/src/plugins/table-column-reordering.jsx

```diff
@@ -66,7 +71,9 @@ export class TableColumnReordering extends React.PureComponent {
 
     if (relativeSourceColumnIndex === -1) return;
 
-    const cellDimensions = this.getCellDimensions();
+    this.getCellDimensions();
+    const { cellDimensions } = this;
```

Comment:
Strange logic. The get* method should return something. Maybe call it cacheCellDimensions?